src/main/java/fi/jubic/quanta/resources/DataConnectionResource.java

```diff
@@ -102,6 +103,28 @@ public DataSeries createDataSeries(
         );
     }
 
+    @PUT
+    @Path("data-series/{id}")
+    public DataSeries updateDataSeries(
```

Comment:
This method needs to be removed. Updating a DataSeries will consist of deleting one DataSeries and creating another one.

I think we can use a query param with the POST request to tell the backend to remove the original DataSeries.